api/priv/repo/seeds.exs

```diff
@@ -12,7 +12,7 @@ db_config = Application.get_env(:api, Api.Repo)
 # that it is adding metadata to the appropriate database
 {output, result} =
   System.shell(
-    "poetry run seed_metadata --env-file ../.env --seed-file tests/test_files/july_17_filepaths.json --clear-static",
+    "poetry run snapshot --env-file ../.env reset",
```

Comment:
reset the database to a blank slate with all of the tables defined using alembic.

it currently takes about 10 seconds to run all the tests locally if we want to seed the db with a snapshot.